backend/src/infra/security/token.manager.ts

```
@@ -59,16 +64,105 @@ export class TokenManager {
     return TokenManager.instance;
   }
 
+  /**
+   * Preload JWT asymmetric keys from the database.
+   */
+  async ensureKeysLoaded(): Promise<void> {
+    if (this.isLoaded) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)